src/main/kotlin/pt/up/fe/ni/website/backend/service/AuthService.kt

```diff
@@ -57,26 +57,45 @@ class AuthService(
         return generateAccessToken(account)
     }
 
+    fun generateRecoveryToken(id: Long): String {
```

Comment:
```suggestion
    private fun generateRecoveryToken(id: Long): String {
```

Reply:
If it's private it will not work, or am I missing something?

Reply:
you're right of course, my bad